production/dpl-workflow.sh

```diff
@@ -11,6 +11,7 @@ source $MYDIR/setenv.sh
 if [ -z $OPTIMIZED_PARALLEL_ASYNC ]; then OPTIMIZED_PARALLEL_ASYNC=0; fi  # Enable tuned process multiplicities for async processing on the EPN
 if [ -z $CTF_DIR ];                  then CTF_DIR=$FILEWORKDIR; fi        # Directory where to store CTFs
 if [ -z $CTF_DICT_DIR ];             then CTF_DICT_DIR=$FILEWORKDIR; fi   # Directory of CTF dictionaries
+if [ -z $CTF_METAFILES_DIR ];        then CTF_METAFILES_DIR="/dev/null"; fi  # Directory where to store CTF files metada, /dev/null : skip their writing
```

Comment:
I would propose that when it becomes operational, we configure it in `/home/epn/pdp/gen_topo.sh` as for the other EPN-related settings.

Reply:
ok, at the moment this is just a placeholder, to pass the (default) option to the workflow